Frameworks/UIKit/UITextField.mm

```diff
@@ -119,6 +94,7 @@ - (void)setText:(NSString*)text {
         _passwordBox.password = [text copy];
```

Comment:
>_passwordBox.password = [text copy]; [](start = 8, length = 36)

It seems like this should move down into the control - we'll just set the Text and it can handle who to set it on (probably via data binding)